cmd/network.go

```diff
@@ -0,0 +1,48 @@
+package cmd
+
+import (
+	"bytes"
+	"context"
+	"fmt"
+
+	"github.com/gridscale/gscloud/render"
+	"github.com/spf13/cobra"
+)
+
+// networkCmd represents the network command
+var networkCmd = &cobra.Command{
+	Use:   "network",
+	Short: "Print network list",
+	Long: `Display a network list as table by default 
+	as json by using the flag --json or -j`,
```

Comment:
No need to doc global flags here. Just write what the subcommand does. E.g., 
Short: "Print networks"
Long: "Print all networks along with their... whatever"

Reply:
changed text